src/Model/ProductRemoteControl.php

```diff
@@ -34,6 +34,8 @@
  *
  * @category Class
  *
+ * @description This product is deprecated.
```

Comment:
Generator should've generate here with PHPDoc `@deprecated` instead of `@description`.